devel/addon/kyverno/BUILD.bazel

```diff
@@ -1,3 +1,21 @@
+load("@io_bazel_rules_docker//container:bundle.bzl", "container_bundle")
+
+container_bundle(
+    name = "bundle_v1.3.6",
+    images = {
+        "ghcr.io/kyverno/kyverno:v1.3.6": "@io_kyverno//image",
+    },
+    tags = ["manual"],
+)
+
+container_bundle(
+    name = "pre_bundle_v1.3.6",
+    images = {
+        "ghcr.io/kyverno/kyvernopre:v1.3.6": "@io_kyverno_pre//image",
+    },
+    tags = ["manual"],
+)
+
```

Comment:
Thanks for doing this. I was too lazy to figure all this out when I introduced kyverno. Sorry for kicking the can down the road.

Reply:
I think we were rushing to release at that point